vector-im/element-x-android
# Manage account and device

## Type of change


## Content


Split settings into 2 entry to manage the account.
<del>Draft until https://github.com/matrix-org/matrix-rust-sdk/pull/2541 is released.</del> Rust upgrade Included!

## Motivation and context


Closes #1303 

## Screenshots / GIFs



## Tests



- Sign in using element.io account
- observe the settings change. The 2 items leads you to the specific page on MAS.

## Tested devices

- OS version(s):

## Checklist
